json-schema-converter/src/test/java/io/confluent/connect/json/JsonSchemaDataTest.java

```diff
@@ -1917,6 +1917,53 @@ public void testOptionalObjectOrArray() {
     Schema connectSchema = jsonSchemaData.toConnectSchema(jsonSchema);
     assertTrue(connectSchema.field("assetMetadata").schema().isOptional());
   }
+
+  @Test
+  public void testIgnoreModernDialects() {
+    String schema = "{ \n"
+        + "  \"$schema\": \"https://json-schema.org/draft/2020-12/schema\",\n"
+        + "  \"type\": \"object\",\n"
+        + "  \"properties\": {\n"
+        + "   \"object_details\": {\n"
+        + "      \"additionalProperties\": true,\n"
+        + "      \"properties\": {\n"
+        + "        \"object_parents\": {\n"
+        + "          \"items\": {\n"
+        + "            \"properties\": {\n"
+        + "              \"object_parents_file_location\": {\n"
+        + "                \"type\": [\n"
+        + "                  \"string\",\n"
+        + "                  \"null\"\n"
+        + "                ]\n"
+        + "              },\n"
+        + "              \"object_parents_id\": {\n"
+        + "                \"type\": [\n"
+        + "                  \"string\",\n"
+        + "                  \"null\"\n"
+        + "                ]\n"
+        + "              }\n"
+        + "            },\n"
+        + "            \"type\": [\n"
+        + "              \"object\",\n"
+        + "              \"null\"\n"
+        + "            ]\n"
+        + "          },\n"
+        + "          \"type\": [\n"
+        + "            \"array\",\n"
+        + "            \"null\"\n"
+        + "          ]\n"
+        + "        }\n"
+        + "      }\n"
+        + "    }\n"
+        + "  }\n"
+        + "}";
+    JsonSchema jsonSchema = new JsonSchema(schema);
+    JsonSchemaData jsonSchemaData =
+        new JsonSchemaData(new JsonSchemaDataConfig(
+            Collections.singletonMap(JsonSchemaDataConfig.IGNORE_MODERN_DIALECTS_CONFIG, "true")));
+    Schema connectSchema = jsonSchemaData.toConnectSchema(jsonSchema);
+    System.out.println(connectSchema);
```

Comment:
I just noticed this - this might have been left in unintentionally?

Reply:
nvm saw [this](https://github.com/confluentinc/schema-registry/pull/3010/files) was filed later